pkg/util/deployment/deployment.go

```diff
@@ -72,7 +72,7 @@ func GetOldReplicaSetsFromLists(deployment extensions.Deployment, c clientset.In
 	// TODO: Right now we list all replica sets and then filter. We should add an API for this.
 	oldRSs := map[string]extensions.ReplicaSet{}
 	allOldRSs := map[string]extensions.ReplicaSet{}
-	rsList, err := getRcList(namespace, options)
+	rsList, err := getRSList(namespace, options)
```

Comment:
`getRcList` (and now `getRSList`) is a local name. That function is passed as an argument to `GetOldReplicaSetsFromLists()` and all the occurrences inside `GetOldReplicaSetsFromLists()` referred to getRcList, so it did not break the tests. But it is still scary. I hope I haven't missed this somewhere else where tests are silently passing.